content/en/docs/reference/glossary/jwt.md

```diff
@@ -0,0 +1,21 @@
+---
+title: JSON Web Token (JWT)
+id: jwt
+date: 2023-01-17
+full_link: https://www.rfc-editor.org/rfc/rfc7519
+short_description: >
+  An authentication token format based on JSON data. JWTs can be digitally
+  signed.
```

Comment:
They can also be encrypted. But they are not an _authentication_ token format, they are "means of representing claims to be transferred between two parties", to quote the linked RFC.

I would remove the second sentence and just replace the original description from the one from the RFC.

Reply:
Done, although I expanded the "more" section to explain how Kubernetes typically uses it